### DISCUSSION TITLE
Remove conda-forge dependency

### DISCUSSION THREAD

**Discussion by anushasekar1** [JuliaPy/CondaPkg.jl#148] [Q&A]
Is there a way to remove the dependency on conda-forge? We have an internal repo that is a proxy for conda-forge and need to use it instead of the regular conda-forge repo. However, CondaPkg.jl seems to always append a -c conda-forge to all environements being created. I have tried setting channel_priority strict in the .condarc and have only the channel I want in the list in .condarc, but that is getting set as the lower priority channel when CondaPkg is resolving environments.

**Comment by cjdoris** [comment]
conda-forge is just the default used when a CondaPkg.toml file does not specify any channels. If you ensure that the channel you want to use is in the CondaPkg.toml for your project and all packages your project depends on, only that channel will be useful. However if you're using PythonCall then you're out of luck because that uses conda-forge.

That said, you say your channel is a proxy. It would be quite straightforward to add an option to CondaPkg to map channels, so that CondaPkg would change any instance of conda-forge to your proxy.

**Comment by anushasekar1** [comment]
It would be awesome to have an option to map channels, our proxy was not named conda-forge, so we had to jump through some hoops to get PythonCall to work again for us. If we have the option to map channels on the developer side, it would make maintenance easier in the future for us!
